Author: aszlig <aszlig@nix.build>
Date:   Wed Oct 10 00:01:51 2018 +0200

    Fix import of Chat model
    
    With the refactor of the chat model to be included within the message
    model the import will fail during migration. So in order to be able to
    migrate older releases, let's fix the import.
    
    Signed-off-by: aszlig <aszlig@nix.build>
    Filename: fix-import-of-chat-model.patch

diff --git a/migrations/groups/migrate-chat.js b/migrations/groups/migrate-chat.js
index 8b341952cd..3b6c717636 100644
--- a/migrations/groups/migrate-chat.js
+++ b/migrations/groups/migrate-chat.js
@@ -8,7 +8,7 @@
  */
 
 import { model as Group } from '../../website/server/models/group';
-import { model as Chat } from '../../website/server/models/chat';
+import { chatModel as Chat } from '../../website/server/models/message';
 
 async function moveGroupChatToModel (skip = 0) {
   const groups = await Group.find({})
